Sources/Segment/Settings.swift

```diff
@@ -47,9 +47,13 @@ public struct Settings: Codable {
     }
     
     public func integrationSettings<T: Codable>(name: String) -> T? {
-        return nil
+        var result: T? = nil
+        guard let settings = integrations?.dictionaryValue else { return nil }
+        if let dict = settings[name], let jsonData = try? JSONSerialization.data(withJSONObject: dict) {
```

Comment:
Not sure if we want to just swallow the throwable errors. We should probably be catching these and logging them at a minimum.